modules/kubernetes/aks-core/k8s-cluster-role.tf

```diff
@@ -20,24 +26,30 @@ resource "kubernetes_cluster_role" "helm_release" {
   }
 }
 
-resource "kubernetes_cluster_role" "toolkit_helm_release" {
+resource "kubernetes_cluster_role" "toolkit_edit" {
   metadata {
-    name = "toolkit-helm-release"
+    name = "toolkit-edit"
+    labels = {
+      "xkf.xenit.io/kind" = "platform"
+    }
   }
   rule {
-    api_groups = ["helm.toolkit.fluxcd.io"]
+    api_groups = ["helm.toolkit.fluxcd.io", "kustomize.toolkit.fluxcd.io", "source.toolkit.fluxcd.io", "notification.toolkit.fluxcd.io"]
     resources  = ["*"]
     verbs      = ["*"]
   }
 }
 
-resource "kubernetes_cluster_role" "toolkit_kustomization" {
+resource "kubernetes_cluster_role" "datadog_edit" {
   metadata {
-    name = "toolkit-kustomization"
+    name = "datadog-edit"
+    labels = {
+      "xkf.xenit.io/kind" = "platform"
+    }
   }
   rule {
-    api_groups = ["kustomize.toolkit.fluxcd.io"]
+    api_groups = ["datadogmetrics.datadoghq.com", "datadogmonitors.datadoghq.com"]
     resources  = ["*"]
-    verbs      = ["get, list, watch"]
+    verbs      = ["*"]
```

Comment:
I thought we agreed to add all rules in the same cluster role? That way we dont need a role binding per rule.

Reply:
Done